platform/linux-generic/pktio/loop.c

```diff
@@ -114,7 +114,7 @@ static int loopback_recv(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 	odp_time_t ts_val;
 	odp_time_t *ts = NULL;
 	int num_rx = 0;
-	int failed = 0;
+	int packets = 0, octets = 0, errors = 0;
```

Comment:
Since all APIs use uint32_t or uint64_t for byte counts, it would be good to use e.g. uint32_t here for octets count. By the C spec, int could be in theory 16 bits.